dara/inc/customizer.php

```diff
@@ -77,7 +77,7 @@ function dara_customize_register( $wp_customize ) {
  * @return interger.
  */
 function dara_sanitize_dropdown_pages( $input ) {
-	if ( is_numeric( $input ) ) {
+	if ( is_numeric( $input ) && $input != get_option( 'page_on_front' ) ) {
```

Comment:
This function is meant to sanitize the input from the dropdown. I think there's an alternative solution for this, I will propose a PR